index.html

```diff
@@ -53,12 +53,14 @@
                             <li><a href="#" onclick="setFrequency(64);">64 Hz</a></li>
                             <li><a href="#" onclick="setFrequency(128);">128 Hz</a></li>
                             <li><a href="#" onclick="setFrequency(256);">256 Hz</a></li>
+                            <li><a href="#" onclick="setFrequency(1024);">1024 Hz</a></li>
                         </ul>
                     </div>
                 </div>
                 <br>
                 <div id="recent-instruction" class="well">The most recent instructions will be shown here when stepping.</div>
                 <hr>
+                <h4><button id="sample-pgrm" class="btn btn-primary">Try a sample program</button></h4>
```

Comment:
Don't think that this feature is necessary and adds additional clutter. There are many tutorials online about how to write a sample MIPS program.